### PR TITLE
Hard-coded header and navigation

### PR DESCRIPTION
**Pull request by helenb**
### What is the context of this PR?

Hard-coded header for the November prototype.
Duplicates the header macro from the design system and replaces `serviceLinks` with `navLinks`
Hard-codes the configuration for the navLinks in `base.html`

### How to review
- Check out the `feature/november-prototype-header` branch
- Open any page in a local build, and check that the header displays as per the designs at desktop and mobile
- Ensure that there is only one link in the header navigation - for now, this has a dummy value. We need to hard-code a link to a bulletin page at some point.

### CODE REVIEW

File: .design-system-version [hunk 1]
```diff
@@ -1 +1 @@
-70.0.8
+70.0.17
```
Comment (helenb) [inline .design-system-version]: This update is now in `main`

File: ons_alpha/jinja2/templates/base.html [hunk 2]
```diff
@@ -67,6 +278,32 @@
     {% endif %}
 {% endblock %}
 
+{% block header %}
+    {{
+    onsHeaderNew({
+    "wide": pageConfig.wide,
+    "fullWidth": pageConfig.fullWidth,
```
Comment (ababic) [inline ons_alpha/jinja2/templates/base.html:285]: Indentation on this section looks a little unclear

File: .design-system-version (hunk 1 above)
Comment (ababic) [reply .design-system-version]: @helenb This change has also been merged into the `november-prototype` branch - so, a rebase should make this redundant.

**Review by ababic — CHANGES_REQUESTED** [review]
@helenb Just a couple of minor considerations from my perspective. Otherwise looking great!

File: ons_alpha/jinja2/templates/base.html (hunk 2 above)
Comment (helenb) [reply ons_alpha/jinja2/templates/base.html:285]: I think this was the linter's doing unfortunately, but I'll check again.

Comment (helenb) [reply ons_alpha/jinja2/templates/base.html:285]: Nah I can't get it to let me indent it nicely. sorry.

**Comment by helenb** [comment]
@ababic I couldn't get the linter to behave with the indenting - are you happy to approve anyway?

**Review by ababic — APPROVED** [review]
LGTM!

**Pull request merged**